src/partials/top-banner.hbs

```diff
@@ -3,10 +3,10 @@
 page.componentVersion.asciidoc.attributes.announcementMessage)
 }}
 <div class="top-banner flex {{#if page.componentVersion.asciidoc.attributes.announcementMessage}}announcement{{/if}}">
-  <div class="container wrapper">
+  <div class="container wrapper flex">
     {{#if page.componentVersion.asciidoc.attributes.announcementMessage}}
+     <img src="{{@root.uiRootPath}}/img/icons/important.svg" alt="">
```

Comment:
nice change. You also need to do the same for the beta site banner (line 13-14 of this file. Basically just switch those lines to get <img> out of <p>)

Reply:
fixed